cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/TimeValuePair.java

```diff
@@ -0,0 +1,67 @@
+package cwms.cda.data.dto.timeseriesprofile;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.data.dto.CwmsDTOBase;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+import java.time.Instant;
+
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class)
+@JsonDeserialize(builder = TimeValuePair.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public final class TimeValuePair extends CwmsDTOBase {
```

Comment:
What is this class for? You already have a `TimeSeriesData` class. Also, this class is time-value-quality so not a pair.... and it replicates the `Record` class in `TimeSeries`

Reply:
Changed out class for Record class within TimeSeries class. Removed this duplicate class